workers/analyzer/docker/Analyzer.Dockerfile

```diff
@@ -223,9 +223,7 @@ RUN . $NVM_DIR/nvm.sh \
     && corepack enable
 
 FROM scratch AS node
-COPY --from=nodebuild $NVM_DIR $NVM_DIR
-# Required for Corepack to dynamically modify binaries of supported package managers.
-RUN sudo chgrp -R 0 /opt/nvm && chmod -R g+rwX /opt/nvm
+COPY --from=nodebuild /opt/nvm /opt/nvm
```

Comment:
Commit message nit: No comma between "anyway, when".

Reply:
> anymore, the reason

I'd just start a new sentence here instead of the comma.

Reply:
> I don't see it being moved, just _re_moved. If `chgrp` is already present in the `components` stage, this should be reworded.

I mixed up `chgrp` and `chmod` in the commit message.